Dockerfile.doc

```diff
@@ -50,6 +50,7 @@ RUN opam depext -uivj 3 \
   jenga \
   logs \
   logs-syslog \
+  lru-cache \
```

Comment:
This thing is called [`lru`](https://github.com/pqwy/lru) since a couple of minutes ago. :smile: